sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/xds/entity/ServiceInstance.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2024-2024 Huawei Technologies Co., Ltd. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.core.service.xds.entity;
+
+/**
+ * service instance interface
+ *
+ * @author daizhenyu
+ * @since 2024-05-09
+ **/
+public interface ServiceInstance {
+    /**
+     * get xds cluster name
+     *
+     * @return cluster name
+     */
+    String getXdsClusterName();
+
+    /**
+     * get service name
+     *
+     * @return service name
+     */
+    String getServiceName();
+
+    /**
+     * get service instance address
+     *
+     * @return service instance address
+     */
+    String getInstanceAddress();
```

Comment:
getHost、getPort.  I think getMetadata is probably neccessary

Reply:
done